graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx

```diff
@@ -2,14 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { components } from 'react-select';
-import { SortableContainer, SortableElement, arrayMove } from 'react-sortable-hoc';
+import { SortableContainer, SortableElement } from 'react-sortable-hoc';
 import { findIndex } from 'lodash';
 
 import Select from 'views/components/Select';
 import styles from './SortableSelect.css';
 
 const SortableValueList = SortableContainer(components.ValueContainer);
 
+const arrayMove = (array, from, to) => {
+  const result = array.slice();
+  result.splice(to < 0 ? array.length + to : to, 0, array.splice(from, 1)[0]);
```

Comment:
result.splice(to < 0 ? array.length + to : to, 0, result.splice(from, 1)[0]);

Reply:
Seems like at least one test case would be nice! :D